src/components/button/Styles.scss

```diff
@@ -1,9 +1,14 @@
 @import '../../styles/utils/colors';
 
-.submitButton{
+.submitButton {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
not sure if grid is the best approach here. does this always keep the same width for the icon space while allowing the text to not affect the icon columns?